packages/angular_devkit/build_angular/BUILD.bazel

```diff
@@ -347,6 +354,12 @@ LARGE_SPECS = {
             "@npm//buffer",
         ],
     },
+    "jest": {
+        "extra_deps": [
+            "@npm//glob",
```

Comment:
I am surprised this worked. But should it `jest` and `jest-environment-jsdom` be added here as well?

Reply:
The unit tests don't actually run Jest because they are testing our code around it (evaluating the `include`  and `exclude` globs for instance). The e2e tests run `npm install jest jest-environment-jsdom` during their setup, so it's outside the Bazel layer.